jck/jck-semiauto-updater/update-check.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
+
+# File to store the previous state
+PREVIOUS_STATE_FILE="/home/jenkins/previous_state.txt"
```

Comment:
Shouldn't assume that the system where this would be run will have a /home/jenkins folder.  If appropriate, would the use of ${WORKSPACE} be better?  Or maybe setting to the location of the script if the file is only intended for temporary use.